components/ProgressBar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { TodoContext } from "../store/store-todo";
+import classes from "../styles/Home.module.css";
+
+const ProgressBar = () => {
+  const todoCtx = useContext(TodoContext);
+
+  const todoList = todoCtx.todoList;
+  return (
+    <div className={classes.progressbox}>
+      <div className={classes.progressgroup}>
+        <div className={classes.progresstitle}>Progress</div>
+        <progress
+          className={classes.progressbar}
+          value={todoList.filter((item) => item.completed === true).length}
```

Comment:
`todoList.filter((item) => item.completed === true).length` is duplicate like the line '19'

so please define it as variable (maybe useMemo)